docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,199 @@
+
+### Critical priority
+
+1. If an issue has been categorized and any of the following criteria apply to it, then the issue should be labeled as critical and must be actively worked on as someone's immediate top priority.
+
+   - Results in any data loss
+   - Critical security or performance issues
+   - Problem that makes a feature unusable
+   - Multiple users experience a severe problem affecting their business, users, etc.
+
+2. Label the issue `priority/critical`.
+3. Escalate the problem to the maintainers.
+4. Assign the issue or ask a maintainer for help assigning the issue to someone as their immediate top priority.
+5. Add the issue to the next upcoming release milestone.
+
+### High priority
+
+1. Label the issue `priority/high`.
+2. Add the issue to the next upcoming release milestone.
+3. Prioritize it or assign someone to work on it now or very soon.
+4. Consider requesting [help from the community](#5-requesting-help-from-the-community).
+
+### Low priority
+
+1. If the issue is deemed possibly useful but is a low priority, then label the issue `priority/low`.
+2. The functional impact of an issue determines how high of a priority the issue is.
+3. Consider requesting [help from the community](#5-requesting-help-from-the-community).
+
+## 5. Requesting help from the community
+
+Due to constraints, maintainers aren't always able to resolve issues in a timely manner, especially those of lower priority. For such issues, consider requesting help from the community. Use your best judgement. In general, requesting help from the community means that a contribution has a good chance of getting accepted and merged.
+
+In many cases, the issue author or the community is most suitable to contribute changes since they're experts in their domain. Also, it's very common for someone to seek help from the community on a problem.
+
+1. Kindly and politely add a comment which will notify the users subscribed to the issue of any updates.
+   - Explain that the issue would be nice to get resolved, but it isn't prioritized to be worked on by maintainers for the foreseeable future.
+   - If possible or applicable, try to help contributors get started by adding pointers to relevant source code or files as well as adding any helpful input on how to resolve the issue.
+2. Label the issue with `help wanted`.
+3. If applicable, label the issue with `beginner friendly` to denote that the issue is suitable for a beginner to work on.
+
+## Investigation of issues
+
+When an issue has all basic information provided, but the reported problem cannot be reproduced at a first glance, the issue is labeled `triage/needs-information`. Depending on the perceived severity and/or number of upvotes, the investigation will either be delegated to another maintainer for further investigation or put on hold until someone else (maintainer or contributor) picks it up and eventually starts investigating it.
```

Comment:
This wording is unclear. I will try to provide an alternate phrasing, but am somewhat guessing at what was intended. I think that if we ask people to label an issue "needs information" because it is difficult to reproduce, even though they do think it has enough information, we should explicitly ask them to explain why they are labeling it as "needs information". 

```
When an issue has all basic information provided, but the reported problem cannot be reproduced at a first glance, label the issue `triage/needs-information` and post a comment explaining why this label is being applied.
```

Reply:
Addressed as suggested